src/apps/transfer_ownership/db/schemas.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, ForeignKey, String
+from sqlalchemy.dialects.postgresql import UUID
+
+from infrastructure.database import Base
+
+
+class TransferSchema(Base):
+    __tablename__ = "transfer_ownership"
+
+    email = Column(String())
+    applet_id = Column(
+        ForeignKey("applets.id", ondelete="RESTRICT"), nullable=False
+    )
+    key = Column(UUID(as_uuid=True))
```

Comment:
In that case, we are duplicating the email field in that database table. For example we use the applet_id which is a foreign key. Could we do the same with user_id instead of the email?

Reply:
I think they can transfer to email who does not have an account in mindlogger.

Reply:
This is the database table.

If we talk about the invitation or something sort of this we can say that we have the public Transfer data model that includes the user's email that does not exist. But there is no reason to have an undefined user's email in the database. Maybe in that case we should pre-create the not-verified user and save the id of that user.

I see the risk: if you enter the wrong email you will have many dead records (never be used) in the database.

Reply:
> Maybe in that case we should pre-create the not-verified user and save the id of that user.
Well, in this case, if user just ignores transfer, we would have dead record in users table. Maybe we should think about having expiry time for transfer ?